code/modules/mob/living/brain/brain_item.dm

```diff
@@ -12,6 +12,7 @@
 	attack_verb = list("attacked", "slapped", "whacked")
 	var/mob/living/brain/brainmob = null
 	var/damaged_brain = 0 //whether the brain organ is damaged.
+	var/decoy_override = 0	//I apologize to the security players, and myself, who abused this, but this is going to go.
```

Comment:
```FALSE``` not 0 (even though it's ```#define FALSE 0``` we enforce this because we're used to having real bools.